MetaMask/metamask-extension
# UX: Ensure multichain native token name is always shown

## Explanation

This PR ensures that the native token name is displayed regardless of the Settings -> Primary currency value.  Previously it was showing "USD" for Ethereum, which is confusing.

* Fixes https://github.com/MetaMask/MetaMask-planning/issues/788

## Screenshots/Screencaps



### Before

![247115557-6cff3806-9310-406f-9a67-afd53b17f486](https://github.com/MetaMask/metamask-extension/assets/46655/33e8e6d9-e21a-4bef-ba4b-e0fb49031028)

### After

<img width="724" alt="SCR-20230621-iwjw" src="https://github.com/MetaMask/metamask-extension/assets/46655/417317ca-aa6f-4685-aeb1-9ca251f6100c">

## Manual Testing Steps

* Flip the settings for Primary currency between Fiat and ETH, see Ethereum shown as the native token in the token list
* Switch to Polygon
* Flip the settings for Primary currency between Fiat and MATIC, see MATIC shown as the native token in the token list

## Pre-merge author checklist


## Pre-merge reviewer checklist


If further QA is required (e.g. new feature, complex testing steps, large refactor), add the `Extension QA Board` label.

In this case, a QA Engineer approval will be be required.
